### PR TITLE
Update links to community chat

### PR DESCRIPTION
**Pull request by Jaykul**
# PR Summary

Add the discord link and switch to aka.ms links that forward directly to open invitations.

## PR Context

The prior link goes to a _login_ form and is not helpful for users.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- x ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by iSazonov** [comment]
@Jaykul Please fix markdown spelling.

**Comment by Jaykul** [comment]
What spelling, @iSazonov?

**Comment by vexx32** [comment]
It's probably triggering on the term 'discord'; you might need to add a spelling exception.

File: README.md [hunk 1]
```diff
@@ -128,9 +128,10 @@ We have a Gitter Room which you can join below.
 
 [![Join the chat at https://gitter.im/PowerShell/PowerShell](https://badges.gitter.im/PowerShell/PowerShell.svg)](https://gitter.im/PowerShell/PowerShell?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-There is also the community driven PowerShell Slack Team which you can sign up for at [Slack].
+The larger community chat is on Discord, Slack and IRC which you can join via [Discord], [Slack] or the #PowerShell channel on Freenode.net!
```
Comment (iSazonov) [inline README.md]: We could reword:
```suggestion
The larger community chat is on [Discord], [Slack] and IRC which you can join via the `#PowerShell` channel on `Freenode.net`!
```

File: .spelling [hunk 2]
```diff
@@ -170,6 +170,7 @@ disable-pswsmancombinedtrace
 disable-runspacedebug
 disable-wsmantrace
 disconnect-pssession
+discord
```
Comment (iSazonov) [inline .spelling]: Perhaps we could remove this if my previous suggestion helps.

**Review by TravisEz13 — APPROVED** [review]

**Review by TravisEz13 — APPROVED** [review]
With the same suggestions as @iSazonov

**Comment by iSazonov** [comment]
@Jaykul Please address last request and we will merge.

**Comment by iSazonov** [comment]
The issue is already fixed in another PR.

**Pull request closed without merging**